index.js

```diff
@@ -2,12 +2,13 @@
 var path = require('path');
 var fs = require('graceful-fs');
 var mkdirp = require('mkdirp');
-var objectAssign = require('object-assign');
+var assign = require('object-assign');
 var onetime = require('onetime');
+var VError = require('verror');
 
 module.exports = function (src, dest, opts, cb) {
 	if (!src || !dest) {
-		throw new Error('`src` and `dest` required');
+		throw new assign(new VError('`src` and `dest` required'), { name: 'CpFileError' })
```

Comment:
An alternative would be:
```js
var err = new VError('`src` and `dest` required');
err.name = 'CpFileError';
throw err;
```
Sadly, `verror` does not provide a shortcut for this.